resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -70,9 +71,15 @@ function task_service {
         else
             task install_source_rpm_systemd
         fi
-    elif [ "$OSFAMILY" = "Debian" ]; then
+      elif [ "$OSFAMILY" = "Debian" ]; then
+        sysv_codenames=("squeeze" "wheezy" "lucid" "precise" "trusty")
+        codename=`facter os.distro.codename`
```

Comment:
The rpm spec directly calls the correct sysv/systemd install.sh function, and I believe previously the debian services were installed automatically by just evaluating the template to the right filename in the debian directory.